src/vasp.jl

```diff
@@ -165,4 +165,59 @@ end
 
 readDOSCAR(filename::AbstractString) = open(readDOSCAR, filename)
 
-readDOSCAR() = open(readDOSCAR, "DOSCAR")
\ No newline at end of file
+readDOSCAR() = open(readDOSCAR, "DOSCAR")
```

Comment:
Try to avoid replacing lines with identical ones like this. (Though to be honest I am not entirely sure how something like this happens, or if it's truly a big deal at the end of the day)